api/src/main/java/io/lumeer/engine/api/dto/SearchSuggestion.java

```diff
@@ -46,7 +49,9 @@ public SearchSuggestion(final String type, final String text, final String icon)
       this(type, text, Collections.emptyList(), icon);
    }
 
-   public SearchSuggestion(final String type, final String text, final List<String> constraints, final String icon) {
+   @JsonCreator
+   public SearchSuggestion(final @JsonProperty("type") String type, final @JsonProperty("text") String text,
+         final @JsonProperty("constraints") List<String> constraints, final @JsonProperty("icon") String icon) {
```

Comment:
Yes because some test throws `nullpointerexception` after changing `response.readEntity(List.class);` to `response.readEntity(new GenericType<List<SearchSuggestion>>(){});` which is (my) preffered way to read data in rest tests and it doesn't show warning..